configs/config_group_1.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Not sure why the change in config file format was necessary, probably due to giving them a name.

Without the need for a name, the grouping could have been derived from previous file format. There was already an object for the definition of a signal, and it would have been easier to add it there.

Example (based `config3.json`)
```
{
    "signals": [
      {
        "path": "Vehicle.Speed",
        "cycle_time": "10ms"
      },
      {
        "path": "Vehicle.TraveledDistance"
        "cycle_time": "1000ms"
      },
      {
        "path": "Vehicle.TraveledDistanceSinceStart"
        "cycle_time": "1000ms"
      }
    ]
  }
```

This is just a hint, because the input comes from DBC, which is defined signal-by-signal. When you want to convert from that input source into the new databroker-perf config file format, you have to implement the "grouping" logic again.

I understand the theoretical idea of having group names though, but there is no such thing as a "group name" for all signals of the same cycle time in DBC. User would have to come up with an artificial name like `All signals with 10ms cycle time`, which I doubt has any benefit.

Reply:
Already discussed -> Groups(signals) represent CAN frames with signals.